cmd/operator-sdk/add/crd.go

```diff
@@ -124,7 +128,8 @@ func verifyCRDFlags() error {
 		return fmt.Errorf("value of --kind must start with an uppercase letter")
 	}
 	if strings.Count(apiVersion, "/") != 1 {
-		return fmt.Errorf("value of --api-version has wrong format (%v); format must be $GROUP_NAME/$VERSION (e.g app.example.com/v1alpha1)", apiVersion)
+		return fmt.Errorf("value of --api-version has wrong format (%v); format must be $GROUP_NAME/$VERSION"+
```

Comment:
Lines like these should be broken by punctuation if possible, ex:
```go
return fmt.Errorf("value of --api-version has wrong format (%v);"+
	" format must be $GROUP_NAME/$VERSION (e.g app.example.com/v1alpha1)", apiVersion)
```

Reply:
ok

Reply:
Hi, 

It is not passing in the CI now because of format. You may fix it with `make format`. See:

<img width="915" alt="Screenshot 2020-01-10 at 08 57 15" src="https://user-images.githubusercontent.com/7708031/72139919-68331080-3387-11ea-8ef1-9dc7e8ff1105.png">

Also, we need to update all similar places with this suggestion as well. Were you able to do that?

Reply:
i found another place and i have made changes, the only issue is that `test-sanity` is no passing. seems like it is checking against previous commit

Reply:
To test locally you need to commit your changes and then, run the make test-sanity.

Reply:
ok done

Reply:
Solved